language/tuning/distilling_step_by_step/Distillation.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>Do you need to install anything for this?</p><p> </p><p>If so, add a section at the top for installation of libraries. You can see an example in this notebook.</p><p> </p><p><a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/language/getting-started/intro_palm_api.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/language/getting-started/intro_palm_api.ipynb</a></p><p> </p><p>Also, be sure to follow the structure of the intro for that notebook. Add the links to Colab/Workbench, list any billable components used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>